RELEASE-NOTES.txt

```diff
@@ -7,6 +7,7 @@
 - [*] Receipts: Email receipts can now be sent to customers after failed payments using WooCommerce Stripe 9.1.0+. [https://github.com/woocommerce/woocommerce-ios/pull/14864].
 - [*] Order Creation: orders are always created using the store's currency, not the user's [https://github.com/woocommerce/woocommerce-ios/pull/14907]
 - [*] Dashboard: Updated the drag gesture on the Performance chart to show stats instantly. [https://github.com/woocommerce/woocommerce-ios/pull/14906]
+- [*] Product Form: Support previewing products on coming-soon stores. [https://github.com/woocommerce/woocommerce-ios/pull/14931]
```

Comment:
nit: I wonder if for end users it's more clear "Product Detail" instead of "Product Form".

Reply:
We'll rewrite the message on the changelog anyway, I think this note is mostly for us developers. Since changing this will trigger CI again, I'll keep it as-is and merge the PR for the code freeze. I hope that's fine.